### PR TITLE
improvement: Make debug completions work for Java files

### PR DESCRIPTION
**Pull request by tgodzik**
Previously, when trying completions in Java files these would hang and not return anything. Now, we complete correctly as in Scala files.

This is not yet super useful, but we should get also expression evaluation for Java files soon.

### CODE REVIEW

**Comment by tgodzik** [comment]
cc @adpi2 This should provide us with Java completions. Although, I see some lower hanging fruits to fix.

**Comment by tgodzik** [comment]
> > cc @adpi2 This should provide us with Java completions.
> 
> Awesome thanks! I'll test that tomorrow.
> 
> > Although, I see some lower hanging fruits to fix.
> 
> What kind of lower hanging fruits?

Related overall to Java completions, we can for example automatically add `;`, which should show us better results.

**Comment by tgodzik** [comment]
Ok, it was a super small issue, I will do the other improvement separately.

**Comment by tgodzik** [comment]
> Ok, it was a super small issue, I will do the other improvement separately.

Doing it in https://github.com/scalameta/metals/pull/5506

File: metals/src/main/scala/scala/meta/internal/metals/Compilers.scala [hunk 1]
```diff
@@ -373,6 +373,12 @@ class Compilers(
                     toDebugCompletionItem(
                       _,
                       adjustStart,
+                      Position.Range(
+                        input,
+                        // account for the added ;
+                        lineStart + 1,
+                        lineStart + 1,
```
Comment (kasiaMarek) [inline metals/src/main/scala/scala/meta/internal/metals/Compilers.scala:380]: why isn't that the range of the currently inserted word?

Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/Compilers.scala:380]: The completions don't use range here, only start and length, so no use setting the end.

Comment (jkciesluk) [reply metals/src/main/scala/scala/meta/internal/metals/Compilers.scala:380]: Actually they use end I think, changing end to 
`lineStart + expressionOffset(expressionText, indentation) + 1` fixes tests

Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/Compilers.scala]: Thanks done! I always changed the input to correspond to the modified source.

**Review by kasiaMarek — APPROVED** [review]

**Pull request merged**